### PR TITLE
XS✔ ◾ Modernized the rule

### PR DESCRIPTION
**Pull request by william-liebenberg**

### CODE REVIEW

File: rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md [hunk 1]
```diff
@@ -44,44 +44,37 @@ Always put "test please" in the email subject so readers know they are expected
...
-3. Specify exactly what is required to be tested by adding some bullet points at the top and highlighting in yellow, so it stands out from the template text, e.g.
-    * Run Timesheet report
-    * Check changing a rate
-4. Make sure the testers send only one bug/suggestion per email.
-5. [Triage](/do-you-send-sprint-forecast-and-sprint-review-retro-emails-to-the-client) emails as they come in for completion in this release, or a later release.
-6. Don't change testers in the middle of a release. It is just sneaky to get a test failed from a tester and then try again by using another tester :-)
-7. Make sure that the testers know which build they are testing. The developers may be 3 builds ahead of the testers, but they need to complete a test run on an individual build to make sure that bugs are fixed and that there are no regressions.
+    **Note:** Having a good branching strategy and using [ephemeral environments](https://www.youtube.com/watch?v=-KrodXD3lPc) makes testing a feature easy and have confidence in the outcome before allowing the code to be committed to `origin/main`. This protects your `origin/main` branch from contamination by code that does not work.
```
Comment (matt-goldman) [inline rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md:55]: Change:
> have confidence

to
gives confidence

File: rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md [hunk 2]
```diff
@@ -44,44 +44,37 @@ Always put "test please" in the email subject so readers know they are expected
...
 
-**Note:** Having a good branching strategy makes this easy as you can run an Internal and External "Test Please" on your DEV branch before allowing the code to be committed to Main/Trunk. This protects your Main/Trunk branch from contamination by code that does not work.
-8. Randomly have the manager do a "Test Please" as well. They give a pass or fail on the job the testers did.
-9. When you receive a "Test Pass" from both testers (and never before), prepare a "Test Please" for the client. (If you are requested to issue an untested release to a client, clearly state "Has not passed internal testing" in the email.)
+6. When you receive a "Test Pass" from both testers (and never before), prepare a "DONE Email" for the client.
```
Comment (matt-goldman) [inline rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md:57]: This is dependent on the DoD

File: rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md [hunk 3]
```diff
@@ -44,44 +44,37 @@ Always put "test please" in the email subject so readers know they are expected
...
+
+    **Note:** If you are requested to issue an untested release to a client, clearly state "Has not passed internal testing" in the email.
```
Comment (matt-goldman) [inline rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md:59]: This should also link to the as per our conversation and for the record rules

**Review by matt-goldman — APPROVED** [review]
I have some suggestions/comments, but happy to approve as is

**Review by pierssinclairssw — APPROVED** [review]

File: rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md (hunk 3 above)
Comment (bradystroud) [reply rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md]: No longer relevant

File: rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md (hunk 2 above)
Comment (bradystroud) [reply rules/conduct-a-test-please-internally-and-then-with-the-client/rule.md]: no longer relevant

**Pull request merged**